core/src/main/scala/org/apache/spark/storage/BlockManagerMasterEndpoint.scala

```diff
@@ -97,6 +97,10 @@ class BlockManagerMasterEndpoint(
     case GetStorageStatus =>
       context.reply(storageStatus)
 
+    case GetCachedBlocks(executorId) => context.reply(getCachedBlocks(executorId))
```

Comment:
Code in next line to follow pattern in other cases.

Reply:
this is removed